boot/freeldr/freeldr/CMakeLists.txt

```diff
@@ -178,12 +177,11 @@ endif()
 add_asm_files(freeldr_common_asm ${FREELDR_COMMON_ASM_SOURCE})
 
 add_library(freeldr_common
+    ${freeldr_common_asm}
     ${FREELDR_BOOTLIB_COMMON_SOURCE}
     ${FREELDR_ARC_COMMON_SOURCE}
     ${FREELDR_NTLDR_COMMON_SOURCE}
-    ${FREELDR_COMMON_SOURCE}
-    ${freeldr_common_asm}
-    linuxboot.c)
+    ${FREELDR_COMMON_SOURCE})
```

Comment:
linuxboot.c is actually redundant since it's already in the `FREELDR_COMMON_SOURCE` list.

Reply:
Moving the asm up, seems to also put it mostly at the beginning of the freeldr_pe.exe, but yeah I agree it's purely decorative. I'll revert that.